boards/arm/mimxrt1064_evk/Kconfig

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (c) 2018, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if BOARD_MIMXRT1064_EVK
+
+choice
+	prompt "Code location selection"
+	default CODE_ITCM
+
+config CODE_ITCM
+	bool "Link code into internal instruction tightly coupled memory (ITCM)"
+
+config CODE_INTERNAL_QSPI
```

Comment:
Why is this 'INTERNAL'?  What is internal about it?

Why aren't we matching how the IMX 105/60 EVKs are done?

Reply:
Since the flash it uses to boot from is internal, not like 105x and 106x which are using the external flash on the board.